app-android-xamarin/Moodis/Moodis/Resources/layout/activity_menu.xml

```diff
@@ -0,0 +1,173 @@
+        android:layout_marginTop="@dimen/grid_unit_9x"
+        android:layout_marginEnd="@dimen/grid_unit_87x"
+        android:layout_marginRight="@dimen/grid_unit_87x"
+        android:text="@string/placeholder"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/lblSadness" />
+
+    <TextView
+        android:id="@+id/lblSadness"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/grid_unit_3x"
+        android:layout_marginEnd="@dimen/grid_unit_87x"
+        android:layout_marginRight="@dimen/grid_unit_87x"
+        android:text="@string/placeholder"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/imageForView" />
+
+    <TextView
+        android:id="@+id/lblSurprise"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/grid_unit_4x"
+        android:layout_marginLeft="@dimen/grid_unit_4x"
+        android:layout_marginEnd="@dimen/grid_unit_56x"
+        android:layout_marginRight="@dimen/grid_unit_56x"
+        android:text="@string/placeholder"
+        app:layout_constraintEnd_toStartOf="@+id/lblHappiness"
+        app:layout_constraintHorizontal_bias="@dimen/gravity_unit_0x"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/lblContempt" />
+
+    <Button
+        android:id="@+id/goToCalendar"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/grid_unit_2x"
+        android:text="@string/calendar_button"
+        android:textSize="@dimen/menu_buttons"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <Button
+        android:id="@+id/playMusic"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/grid_unit_6x"
+        android:layout_marginLeft="@dimen/grid_unit_6x"
+        android:layout_marginBottom="@dimen/grid_unit_2x"
+        android:text="@string/play_music_button"
+        android:textSize="@dimen/menu_buttons"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/goToCalendar" />
+
+    <Button
+        android:id="@+id/groups"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/grid_unit_5x"
+        android:layout_marginLeft="@dimen/grid_unit_5x"
+        android:layout_marginBottom="@dimen/grid_unit_2x"
+        android:text="@string/groups_button"
+        android:textSize="@dimen/menu_buttons"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/StopMusic" />
```

Comment:
Add constraint end to end of parent. Because it is missing, on some devices this button gets cut off.